tests/assets/wpt/README.md

```diff
@@ -1,10 +1,12 @@
-Web platform tests are copied from https://github.com/web-platform-tests/wpt.
+Web platform tests are copied from https://github.com/web-platform-tests/wpt at revision `5adfe4e8cd223729aa2942915e8a515c079ed0ef`.
```

Comment:
nit: a `roll.sh` script would be nice there (which does e.g. git clone, rm -rf .git, only keep the relevant folder). Since its only a subfolder?

Reply:
So far I did it once, so I'll wait for the second time 😄